client/me/security-checkup/recovery-email-validation-notice.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Notice from 'components/notice';
+import NoticeAction from 'components/notice/notice-action';
+
+class RecoveryEmailValidationNotice extends Component {
```

Comment:
Yes, you read my mind: https://github.com/Automattic/wp-calypso/blob/582713f831cde99141e10e52a36be6466adfa3ea/client/me/security-checkup/recovery-phone-validation-notice.jsx

These two are quite different. The phone one contains a form for inputing validation code, and thus contains extra components and needs a few more action creators. At the moment, I couldn't see an obvious way to combine the two. How do you think? If you got any idea, I'd love to try :)